cougarvision_utils/get_images.py

```diff
@@ -68,11 +70,21 @@ def request_strikeforce(username, auth_token, base, request, parameters):
         strikeforce
     '''
     call = base + request + "?" + parameters
```

Comment:
This should be a function that is called, it shouldn't just run every time the module is opened